apps/platform-app/src/components/hunter-landing/HunterHeader/HunterHeader.tsx

```diff
@@ -1,7 +1,67 @@
+'use client'
+import { LocaleSwitcher } from '@/app/[locale]/(profile)/(components)/LocaleSwitcher/LocaleSwitcher'
+import { I18nNamespaces } from '@/i18n/request'
+import { AppRoutes } from '@/utils/routes'
+import { Button } from '@gdh/ui-system'
+import { useTranslations } from 'next-intl'
+import { useRouter } from 'next/navigation'
+import { useState } from 'react'
+import { LogoLight } from '../UI/LogoLight/LogoLight'
 import styles from './HunterHeader.module.scss'
 
-type Props = {}
+export const HunterHeader = () => {
+  const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false)
+  const [menuVisible, setMenuVisible] = useState(false)
+  const t = useTranslations(I18nNamespaces.HunterHeader)
+  const router = useRouter()
 
-export const HunterHeader = (props: Props) => {
-  return <div className={styles.wrapper}> HunterHeader</div>
+  const handleAddJob = () => {
+    router.push(AppRoutes.postJob)
+  }
+
+  const toggleMobileMenu = () => {
+    if (isMobileMenuOpen) {
+      setMenuVisible(false)
+      setTimeout(() => {
+        setIsMobileMenuOpen(false)
+      }, 300)
+    } else {
+      setIsMobileMenuOpen(true)
+      setTimeout(() => {
+        setMenuVisible(true)
+      }, 10)
+    }
+  }
+
+  return (
+    <section id="HunterHeader" className={styles.wrapper}>
```

Comment:
Id HunterHeader in section tag - it is a little confusing
Btw is better to put that into <Container/> component to avoid problems with display on huge screens
![image](https://github.com/user-attachments/assets/f0401171-0c2f-4634-88c1-36ac67d6151a)